NEWS.md

```diff
@@ -1,4 +1,9 @@
-# teal.widgets 0.4.1.9010
+# teal.widgets 0.4.2
+
+### Enhancements
+* Introduce `nested_closeable_modal()` that can create nested popups inside an already existing popup.
```

Comment:
For the record, I have some doubts about this feature that I expressed after it was added and that were not addressed.

Reply:
Can you open an issue to express your concern, ensuring that we don't overlook it? 
We added the `experimental` label to this function. If we miss anything, we can update it in an iterative manner.

Reply:
See [here](https://github.com/insightsengineering/teal.widgets/pull/199#issuecomment-1768133369) and below.

Reply:
Thanks, I created an issue for this.
https://github.com/insightsengineering/teal.widgets/issues/214